.github/actions/terraform-init/action.yml

```diff
@@ -0,0 +1,39 @@
+name: "Initiate Terraform"
+description: "Initiate Terraform"
+inputs:
+  S3TerraformBucket:
+    description: "Bucket name of S3 bucket to use"
+    required: true
+  S3TerraformKey:
+    description: "S3 key of terraform tfstate file"
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - name: Install Terraform
+      uses: hashicorp/setup-terraform@v3
+
+    - name: Terraform backend
+      shell: bash
+      run: |
+        cat <<EOF > .infra/infra/backend_override.tf
+        terraform {
+          backend "s3" {
+          }
+        }
+        EOF
+
+    - name: Terraform init
+      shell: bash
+      run: |
+        terraform init \
+          -backend-config=bucket=${{ inputs.S3TerraformBucket }} \
+          -backend-config=key=${{ inputs.S3TerraformKey }} \
+          -backend-config=region=${AWS_REGION} \
```

Comment:
Where do these values come from?

Reply:
I assume those values are already available. So they are given when you call the action via the `env` entry. But in our case each workflow already execute an OIDC action which will set those envvars